### PR TITLE
Make changes as per suggestion

### PR DESCRIPTION
**Pull request by Ashish-srij**
Made the changes as per your suggestion

### CODE REVIEW

File: stories/Organism/FilterAndSearchBar/FilterAndSearchBar.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+export const FilterAndSearchBar = () => {
+  useEffect(() => {
+    toggleFilter();
+  }, []);
+
+  return (
+    <div className="search_filter">
+      <button className="toggleSearch">
+        {' '}
```
Comment (surbhi90) [inline stories/Organism/FilterAndSearchBar/FilterAndSearchBar.jsx:16]: @Ashish-srij can you remove this extra space if not required

File: stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
+@import '../../assets/scss/mixins';
+@import '../../assets/scss/breakpoints';
+@import '../../assets/scss/variables';
+
+.toggleSearch {
+  background: none;
+  border: 0;
+  border-bottom: 1px solid #000;
```
Comment (surbhi90) [inline stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss:8]: @Ashish-srij please use the variable

File: stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  font-size: $font-size-16;
+}
+
+[dir='rtl'] {
+  .Selected_chips {
+    .chips {
+      margin: 0 0 $spacing-12 $spacing-12;
+    }
+  }
+}
```
Comment (surbhi90) [inline stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss:51]: @Ashish-srij Seems like you have not linted the code so please fix the indentation

File: stories/Molecules/Multiselect/MultiSelect.jsx [hunk 4]
```diff
@@ -3,15 +3,15 @@ import './multiselect.scss';
...
+const SelectTag = ({ text, eleId }) => {
   useEffect(() => {
     init();
   }, []);
   return (
     <div className="multi-select regions-select" data-multi-select="">
       <button
         className="select-control"
-        id="select_button"
+        // id="select_button"
```
Comment (surbhi90) [inline stories/Molecules/Multiselect/MultiSelect.jsx:14]: @Ashish-srij Please remove this commented code

File: stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+.clear_chips {
+  border-bottom: 1px solid $color-red;
+  cursor: pointer;
+  font-size: $font-size-16;
+}
+
+[dir='rtl'] {
+  .Selected_chips {
```
Comment (surbhi90) [inline stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss:46]: @Ashish-srij please follow the proper classnames naming convention as captitalization and underscores are not allowed

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

**Review by Ashish-srij — COMMENTED** [review]
Update as per feedback.

File: stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+  &.show-filter {
+    display: block;
+  }
+  .multi-select { padding-top: 24px; }
+}
+
+#selectedchips {
+  padding-top: 24px;
+
```
Comment (surbhi90) [inline stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss:25]: @Ashish-srij Please don't use much id's for the styling as we should use classes for styling and id's for JS

File: stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss [hunk 7]
```diff
@@ -0,0 +1,51 @@
+@import '../../assets/scss/mixins';
+@import '../../assets/scss/breakpoints';
+@import '../../assets/scss/variables';
+
+#toggleSearch {
```
Comment (surbhi90) [inline stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss:5]: @Ashish-srij Please don't use the camelcasing in class and id names

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss (hunk 6 above)
Comment (Ashish-srij) [reply stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss:25]: Done

File: stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss (hunk 7 above)
Comment (Ashish-srij) [reply stories/Organism/FilterAndSearchBar/FilterAndSearchBar.scss]: Done

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**